runtime/astar/src/lib.rs

```diff
@@ -1258,6 +1258,7 @@ pub type Executive = frame_executive::Executive<
 >;
 
 parameter_types! {
+    // percentages below are calulated based on total issuance at the time when dApp staking v3 was launched (8.4B)
```

Comment:
Might be worth considering adding more checks for this into the try-runtime check 🙂 

For another PR though.

Reply:
Are you thinking of sanity checks in `pre_upgrade`, like:

```rs
#[cfg(feature = "try-runtime")]
fn pre_upgrade() -> Result<Vec<u8>, TryRuntimeError> {
    let tier_thresholds: Result<BoundedVec<TierThreshold, T::NumberOfTiers>, _> =
        BoundedVec::try_from(TierThresholds::get().to_vec());
    assert!(tier_thresholds.is_ok());

    ...
}
```

Reply:
Both pre & post.

E.g. take current thresholds (active ones) in pre-upgrade, send them to post upgrade.

In post-upgrade, after you derive the new thresholds, ensure they are e.g. within 10% of the old ones.